packages/amplify-category-function/src/provider-utils/supportedServicesType.ts

```diff
@@ -2,21 +2,29 @@ import { FunctionParameters } from 'amplify-function-plugin-interface';
 
 export type SupportedServices = Record<Service, ServiceConfig>;
 
+export enum Service {
```

Comment:
we shouldn't have Service defined here and ServiceNames defined elsewhere as the same thing. One should be defined and the other reference it. Probably just delete this one if you go with the above parameterization of SupportedServices